allure-testng-adaptor/src/main/java/ru/yandex/qatools/allure/testng/AllureTestListener.java

```diff
@@ -212,14 +262,22 @@ String getCurrentSuiteTitle(ITestContext iTestContext) {
         return suite + " : " + xmlTest + params;
     }
 
-    private String getCurrentSuitePrefix(ITestResult iTestResult) {
-        return "{" + getCurrentSuiteTitle(iTestResult.getTestContext()) + "}";
-    }
-
     private void fireFinishTest() {
         getLifecycle().fire(new TestCaseFinishedEvent());
     }
 
+    private void fireTestCaseCancel(ITestResult iTestResult) {
+        Throwable skipMessage = new Throwable() {
```

Comment:
The reason for skipped or cancelled method it's just a message that saying that somewhere else in your test there was en event which maybe an exception which cause this particular method to be skipped. There is no reason to throw another exception  rather to provide descriptive message. As workaround I simulated this message by throwing anonymous exception to provide just message like this: ":Skipped due to dependency on other skipped or failed test methods"